UniMath/README.md

```diff
@@ -72,6 +74,15 @@ less fragile and to make the files have a more uniform and pleasing appearance.
   abstracted lemma (whose name typically ends with `_subproof`), because
   its type may vary from one version of Coq to another.  Coq's current behavior is also
   unlikely to be duplicated precisely by a future proof assistant. 
+* Define and use accessor functions for structures instead of chains
+  of `pr1` and `pr2`. This makes the code easier to maintain in the
+  long run (if the structure is rearranged the proofs will still work
+  if the accessor functions are changed accordingly).
+* Define constructor functions starting with `mk_` for structures
```

Comment:
Another way to get a flatter proof script with with `repeat split` or `repeat use tpair`, so why is this proposed guideline important?

We currently have an open proposal (#176) to come up with a standard way to construct identifier names,
and this paragraph takes a stance in opposition to the proposal by requiring the form `mk_S`, as opposed to `makeS`.

Reply:
I think `use mk_pullback` is more informative than `repeat split`, so I definitely prefer the former. 

I have been consistently using `mk_` for naming constructors in CategoryTheory. If the proposal you refer to is adopted then we should rename all of these. For now I'll just remove "starting with `mk_`" from this sentence. The important point is not what the constructors are called but that they should exist.